src/components/molecules/Pagination/index.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="pagination">
+    <ol class="pagination-list">
+      <li class="pagination-list-btn">
+        <app-router-link
+          :to="path"
+          white
+          class="pagination-list-btn_link"
+          :class="currentPage === 1 ? 'pagination-list-btn_disabled' : ''"
```

Comment:
[imo]
1行にまとめられそうです！

```suggestion
          :class="[
            'pagination-list-btn_link',
            {'pagination-list-btn_disabled': currentPage === 1 }
          ]"
```